go.mod

```diff
@@ -7,6 +7,7 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.0+incompatible // oras dep requires a replace is set
 	github.com/docker/docker => github.com/docker/docker v20.10.9+incompatible // oras dep requires a replace is set
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.7
+	github.com/harvester/node-manager => github.com/Vicente-Cheng/node-manager v0.0.0-20230511091905-e09f64949031
```

Comment:
should we switch to the latest tag for `harvester/node-manager`

Reply:
Yes, I will switch back once the node-manager PR is merged